pkg/storage/sqlite/schema/schema.sql

```diff
@@ -0,0 +1,96 @@
+CREATE TABLE IF NOT EXISTS "indexer" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "name" TEXT NOT NULL,
+    "priority" INTEGER NOT NULL DEFAULT 25,
+    "uri" TEXT NOT NULL,
+    "api_key" TEXT
+);
+
+CREATE TABLE IF NOT EXISTS "quality_definition" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "quality_id" INTEGER NOT NULL,
+    "name" TEXT NOT NULL,
+    "preferred_size" NUMERIC NOT NULL,
+    "min_size" NUMERIC NOT NULL,
+    "max_size" NUMERIC NOT NULL,
+    "media_type" TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "quality_profile" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "name" TEXT NOT NULL,
+    "cutoff_quality_id" INTEGER NOT NULL,
+    "upgrade_allowed" BOOLEAN NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "profile_quality_item" (
```

Comment:
This reads kind of odd to me with it being the only one with profile first vs quality.

Reply:
yeah I agree, I noticed that myself and ended up not changing it initially. Swapped it to quality_profile_item instead now